server/tm_cmds.go

```diff
@@ -123,3 +127,67 @@ func VersionCmd() *cobra.Command {
 		},
 	}
 }
+
+// makeKeyMigrateCmd is ported from tendermint's key-migrate command, but
+// uses the SDK's own server.Context.
+// ref: https://github.com/tendermint/tendermint/blob/master/UPGRADING.md#database-key-format-changes
+func makeKeyMigrateCmd() *cobra.Command {
```

Comment:
this is called from cosmovisor as well right?

Reply:
it should, but it's not yet. We need a separate PR for the cosmovisor updates, as this PR is intended to be backported.

I can create an issue for cosmovisor.

Reply:
#12039